packages/plugins/other/visitor-plugin-common/tests/enum-values.spec.ts

```diff
@@ -147,4 +147,55 @@ describe('enumValues', () => {
       },
     });
   });
+
+  const schemaWithNonStringEnumValues = new GraphQLSchema({
+    query: new GraphQLObjectType({
+      name: 'Query',
+      fields: {
+        test: {
+          type: new GraphQLEnumType({
+            name: 'Test',
+            values: {
+              A: {
+                value: 1,
+              },
+              B: {
+                value: true,
+              },
+              C: {
+                value: null,
+              },
+              D: {
+                value: undefined,
+              },
+            },
+          }),
+        },
+      },
+    }),
+  });
+
+  it('should respect non-string enum values', () => {
+    const result = parseEnumValues({
+      schema: schemaWithNonStringEnumValues,
+      mapOrStr: {},
+      ignoreEnumValuesFromSchema: false,
+    });
+
+    expect(result).not.toEqual({
```

Comment:
Oh, sorry about this. this test case doesn't make sense

Reply:
@cometkim could you create another PR to fix that? :)